seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/sink/PaimonSinkFactory.java

```diff
@@ -56,7 +56,8 @@ public OptionRule optionRule() {
                         PaimonSinkConfig.DATA_SAVE_MODE,
                         PaimonSinkConfig.PRIMARY_KEYS,
                         PaimonSinkConfig.PARTITION_KEYS,
-                        PaimonSinkConfig.WRITE_PROPS)
+                        PaimonSinkConfig.WRITE_PROPS,
+                        PaimonConfig.HADOOP_CONF)
```

Comment:
Why not add `PaimonConfig.HADOOP_CONF_PATH`?

Reply:
> Why not add `PaimonConfig.HADOOP_CONF_PATH`?

Sorry, I'll fix it.